src/presenter/presenter.py

```diff
@@ -157,6 +157,12 @@ def init_module_custom_connections(self, module):
         elif module.type == "Load":
             module.parameters.browse.clicked.connect(lambda: self.browse_path(module))
 
+        elif module.type == "ExtractChannels":
+            module.parents[0].displayed.connect(lambda: self.init_module_custom_connections(module))
+            parent_im = module.getData(module.get_parent_name())
+            if isinstance(parent_im, np.ndarray) and parent_im.ndim == 3:
+                module.parameters.alpha.setVisible(parent_im.shape[2] == 4)
```

Comment:
Do we really want to handle the alpha channel, I thought we would not!?

Reply:
ok! I thought we keep it if alpha channel is not  always the same value..

Reply:
So we don't take alpha channel into account at all! deal